docs/embedding/full-app-embedding.md

```diff
@@ -4,161 +4,227 @@ redirect_from:
+Metabase offers several [types of embedding](./introduction.md) with different levels of customization and security.
 
-You'll be putting the whole Metabase app into an iframe, and the SSO integration you've set up with Metabase will be used to make sure the embedded Metabase respects the collection and data permissions you've set up for your user groups. Clicking on charts and graphs in the embed will do just what they do in Metabase itself. You can even display a specific Metabase collection in an embed to allow your users to browse through all the dashboards and questions that you've made available to them. The only difference is that Metabase's top nav bar and global search will not be rendered in your iframe.
+**Full-app embedding** is the only type of embedding that integrates with your [permissions](../permissions/introduction.md) and [SSO](../people-and-groups/start.md#authentication) to give people the right level of access to [query](https://www.metabase.com/glossary/query_builder) and [drill-down](https://www.metabase.com/glossary/action_menu) into your data.
 
-## What you'll be doing
+## Prerequisites
 
-To get this going, you're going to need:
+1. Make sure you have a [license token](../paid-features/activating-the-enterprise-edition.md) for a [paid plan](https://store.metabase.com/checkout/login-details).
+2. Organize people into Metabase [groups](../people-and-groups/start.md).
+3. Set up [permissions](../permissions/introduction.md) for each group.
+4. Set up [SSO](../people-and-groups/start.md#authentication) to automatically apply permissions and show people the right data upon sign-in.
 
-- A [paid plan of Metabase](https://www.metabase.com/pricing) that includes full-app embedding.
-- A separate web application that you want to embed your dashboards and charts in.
+If you're dealing with a [multi-tenant](https://www.metabase.com/learn/customer-facing-analytics/multi-tenant-self-service-analytics) situation, check out our recommendations for [Configuring permissions for different customer schemas](https://www.metabase.com/learn/permissions/multi-tenant-permissions).
 
-## Enabling embedding in Metabase
+## Enabling full-app embedding in Metabase
 
-First, let's enable embedding in your Metabase instance. Go to the Admin Panel, and under Settings, go to the “Embedding in other applications” tab. From there, click “Enable.”
+1. Go to **Admin settings** > **Embedding**.
+2. Click **Enable**.
+3. Click **Full-app embedding**.
+4. Under **Authorized origins**, add the URL of the website or web app where you want to embed Metabase.
```

Comment:
```suggestion
4. Under **Authorized origins**, add the URL of the website or web app where you want to embed Metabase (e.g., `https://*.example.com`).
```
We do mention the example in the field itself; the issue is that they might paste an incorrectly formatted string in there first, and may miss the example.